cmd/monaco/main.go

```diff
@@ -15,12 +15,54 @@
 package main
 
 import (
-	"os"
-
+	"context"
+	"fmt"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/featureflags"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/version"
+	monacoVersion "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/version"
+	"net/http"
+	"os"
 )
 
 func main() {
+	var versionNotification string
+	if !featureflags.SkipVersionCheck().Enabled() {
+		go setVersionNotificationStr(&versionNotification)
+	}
+
 	statusCode := runner.Run()
+	notifyUser(versionNotification)
 	os.Exit(statusCode)
 }
+
+func setVersionNotificationStr(msg *string) {
+	currentVersion, err := version.ParseVersion(monacoVersion.MonitoringAsCode)
+	if err != nil {
+		log.Debug("Could not perform version check: %s", err)
```

Comment:
nit:
Intetion: Differentiate between the local check and the remote one. 
For us, the local one will always fail due to being a 'dev' version.

```suggestion
		log.WithFields(field.Error(err)).Debug("Can't parse current monaco version: %s", err)
```